Source/Android/app/src/main/res/layout/activity_settings.xml

```diff
@@ -1,5 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-             android:layout_width="match_parent"
-             android:layout_height="match_parent"
-             android:id="@+id/frame_content"/>
+<androidx.coordinatorlayout.widget.CoordinatorLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/coordinator_main"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/appbar_settings"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true">
+
+        <com.google.android.material.appbar.MaterialToolbar
+            android:id="@+id/toolbar_settings"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="?attr/colorSurface"
+            app:layout_scrollFlags="scroll|enterAlways|snap"/>
```

Comment:
Is there a particular reason for having the behavior where this hides when scrolling down? Personally I find that behavior a little bit annoying with how tall phones are nowadays, but if there's some particular reason for it (like if it's standard in material design), I suppose it's okay.

Reply:
You can do either really. As long as the expanded app bar gets a drop shadow when scrolling, it stays within spec.

Reply:
I'm gonna try to get a collapsing toolbar layout to maybe improve the situation on taller phones

Example -
![image](https://user-images.githubusercontent.com/14132249/184548838-df45f22c-8c14-4b97-b25b-25e4d2694930.png)

Reply:
*Correction, you don't need the drop shadow to appear to stay in spec. It's not necessary anymore.

Reply:
Ok, I have the new app bar implemented I just need an opinion. Would you prefer the medium or large collapsing top app bar?

Medium - 
![image](https://user-images.githubusercontent.com/14132249/184756404-14389bb6-c0f8-4d82-8839-70365eafa00e.png)

Large -
![image](https://user-images.githubusercontent.com/14132249/184756364-33fca148-fdb1-47b1-a9fa-2fa54d1f0dca.png)

Reply:
Does the large one just have more blank space? What's the point of that?

Reply:
More blank space and a slightly larger title. It brings all of the settings options closer to the bottom for better reachability.

Reply:
Also it collapses and doesn't hide the top bar anymore